src/main/java/Task.java

```diff
@@ -0,0 +1,38 @@
+public class Task {
+    String task;
```

Comment:
Might give the impression that the member field 'task' is an instance of Task. Could use a better name ( eg taskDescription )